src/CalcManager/Ratpack/ratpak.h

```diff
@@ -148,6 +148,9 @@ extern PRAT rat_min_i32;
 #define LOGNUM2(pnum) ((pnum)->cdigit+(pnum)->exp)
 #define LOGRAT2(prat) (LOGNUM2((prat)->pp)-LOGNUM2((prat)->pq))
 
+// SIGN returns the sign of the rational
+#define SIGN(prat) ((prat)->pp->sign*(prat)->pq->sign)
```

Comment:
Hmm, I realize this is the style of this file, but maybe we should still update this to be a free function?

Reply:
I'm not a big fan of macros, I would prefer `Sign()` to be a function of (P)RAT, but we should keep the code consistent. (Future PR to refactor RatPack?)